templates/sounds/sound.html

```diff
@@ -41,7 +41,7 @@
 
 {% block section_content %}
 
-{% cache 3600 sound_header sound.id request.user.is_authenticated request.user.profile.is_adult %}
+{% cache 3600 sound_header sound.id is_explicit %}  {# cache both blurred and normal version for explicit sounds #}
```

Comment:
Did you make sure that `request.user.is_authenticated` is not used for other things that might affect the template?